draft-ietf-tls-dtls-connection-id.md

```diff
@@ -506,8 +519,8 @@ TBD1    connection_id   -        Y          N           [[This doc]]
 
 Note: The value "N" in the Recommended column is set because this 
 extension is intended only for specific use cases. This document describes 
-an extension for DTLS 1.2 only; it is not to TLS (1.3). The DTLS 1.3 
-functionality is described in {{I-D.ietf-tls-dtls13}}. 
+an extension for DTLS 1.2 only; it is not applicable to TLS or DTLS 1.3.
+Analogous functionality for DTLS 1.3 is described in {{I-D.ietf-tls-dtls13}}. 
```

Comment:
This text actually seems like it's OBE, as DTLS 1.3 uses the same extension.

Reply:
Some further updates pushed here.  It seems simplest to put the 'CH,SH' in the TLS 1.3 column in this document rather than have dtls13 modify the registration, but there's also a decent argument for keeping all the 1.3 stuff out of this document.  (So I made it as a separate commit that is easy to remove if needed.)

Reply:
I don't care about the direction we take here. If you think it is easier for IANA to register the value with this document then that's fine for me.